components/director/pkg/resource/resource.go

```diff
@@ -20,3 +20,15 @@ const (
 	AutomaticScenarioAssigment Type = "AutomaticScenarioAssigment"
 	Webhook                    Type = "Webhook"
 )
+
+type SQLOperation string
```

Comment:
This type and the constants below belong more to the persistence/const.go

Reply:
discussed offline